config_panel.toml

```diff
@@ -0,0 +1,16 @@
+version = "1.0"
+
+[bind]
```

Comment:
What you declare here is an ID for the `"Mautrix Signal configuration"` panel. What I understood of the config_panel.toml file is that the first level ID is only meant to be meaningful when using it in the CLI.

You may use a different ID if you want to. Like `[main]` or `[bridge]` (and consequently adapt the IDs below).

Reply:
Thanks for the explanation, I better understood.